src/libs/tools/src/plugindatabase.cpp

```diff
@@ -367,5 +382,12 @@ std::string MockPluginDatabase::lookupInfo (PluginSpec const & spec, std::string
 
 	return "";
 }
+
+PluginDatabase::func_t MockPluginDatabase::getSymbol (PluginSpec const & spec ELEKTRA_UNUSED,
+						      std::string const & which ELEKTRA_UNUSED) const
+{
+	// TODO implement mockup
+	return NULL;
+}
```

Comment:
For unit tests you should return some function as done for other mocks with `data`. Then you can test if the mounting process fails/succeeds as desired with different checkplugin configs. There should be at least 3 unit tests: with the 3 return values checkconf has.